command/plugin_status_csi.go

```diff
@@ -183,6 +189,10 @@ func (c *PluginStatusCommand) formatControllerCaps(controllers map[string]*api.C
 func (c *PluginStatusCommand) formatNodeCaps(nodes map[string]*api.CSIInfo) string {
 	caps := []string{}
 	for _, node := range nodes {
+		// TODO: move this up to top-level k/v?
```

Comment:
worth making a ticket?

Reply:
Oh, actually I meant to remove it. Thanks.